pytrip/tripexecuter/plan.py

```diff
@@ -491,7 +481,7 @@ def _make_exec_fields(self):
             line += " doseext({:.4f})".format(_field.dose_extension)
             line += " contourext({:.2f})".format(_field.contour_extension)
             line += " zsteps({:.3f})".format(_field.zsteps)
-            line += ' proj({:s})'.format(_field.projectile)  # TODO: check if 'C' or better '12C"
+            line += ' proj({:s})'.format(_field.kernel.projectile.iupac)  # TODO: check if 'C' or better '12C"
```

Comment:
add a comment was is iupac, providing an example ('H' or 'H1' or '1H' ? - what is it ?)

Reply:
There is a comment in Projectile class:
https://github.com/pytrip/pytrip/blob/00cec4cd960f27193966d3f31d2b137b6a996fd7/pytrip/tripexecuter/projectile.py#L45